src/tests/CMakeLists.txt

```diff
@@ -16,6 +16,8 @@ endif()
 add_subdirectory(ngraph_helpers)
 add_subdirectory(unit)
 add_subdirectory(ie_test_utils)
+add_subdirectory(util)
+add_subdirectory(engines_util)
```

Comment:
@iefode we need to decrease the number of utility libraries 😄

Reply:
What is `util` and `engine_util`? Why they are out of `ie_test_utils`? we can move theam to the dir and rename it quite later

Reply:
@ilyachur The question to you